components/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/bean/SSOAgentConfig.java

```diff
@@ -742,6 +745,29 @@ public void checkServerTrusted(java.security.cert.X509Certificate[] certs,
         private String relayState = null;
         private String signatureValidatorImplClass = null;
         private int timeStampSkewInSeconds = 300;
+        private String artifactResolveURL = null;
+        private boolean enableArtifactResolveSigning;
+
+        public void setArtifactResolveURL(String artifactResolveURL) {
```

Comment:
Add java doc comments for all the public methods.

Reply:
There are simple getters and setters. Should we have doc comments to these as well?

Reply:
Yes, these are public methods.